tests/TestCase/Shell/Task/MigrationSnapshotTaskTest.php

```diff
@@ -60,12 +60,14 @@ public function setUp()
     public function testNotEmptySnapshot()
     {
         $this->Task->params['require-table'] = false;
+        $this->Task->params['connection'] = 'test';
+        $this->Task->params['plugin'] = 'BogusPlugin';
 
         $version = Util::getCurrentTimestamp();
 
         $this->Task->expects($this->once())
             ->method('dispatchShell')
-            ->with('migrations', 'mark_migrated', $version);
+            ->with('migrations mark_migrated ' . $version . ' -c test -p BogusPlugin');
```

Comment:
This test is a bit fragile because of the timestamp. You can use something like:

``php
->with(
  $this->logicalAnd($this->stringContains('migrations mark_migrated'),
  $this->logicalAnd($this->stringContains('-c test -p BogusPlugin')
));
```

Will avoid this issue.